src/Components/Modal/modal.css

```diff
@@ -0,0 +1,66 @@
+.modal {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width:100%;
+    height: 100%;
+    background: rgba(0, 0, 0, 0.6);
+  }
+  
+  .modal-main {
+    position:fixed;
+    background: white;
+    width: 40%;
+    height: auto;
+    top:50%;
+    left:50%;
+    transform: translate(-50%,-50%);
+    border-radius: 1.5%;
+  }
+  
+  .display-block {
+    display: block;
+  }
+  
+  .display-none {
+    display: none;
+  }
+  
+  button {
+    margin: 20px 100px 10px 100px;
+    width: 70px;
```

Comment:
The design looks so simple, kindly enhance it.
Functionality LGTM
Kindly work on UI for Modal.